### PR TITLE
Added missing compiler flags for Intel c++, which broke the build in …

### PR DESCRIPTION
**Pull request by rainergericke**
…release mode. Please check it.

### CODE REVIEW

**Comment by LebedevRI** [comment]
This should be fine, except for the cla bot.

**Comment by dominichamon** [comment]
@rainergericke can you satisfy the CLA bot please?

File: CMakeLists.txt [hunk 1]
```diff
@@ -219,6 +219,16 @@ else()
   add_cxx_compiler_flag(--coverage COVERAGE)
 endif()
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel" AND "${CMAKE_BUILD_TYPE}" STREQUAL "Release")
+	# Handle some issues: 
+	# 1. implicit type casting not allowed (sysinfo.cc)
```
Comment (LebedevRI) [inline CMakeLists.txt]: also, tabs; should be spaces

File: CMakeLists.txt [hunk 2]
```diff
@@ -219,6 +219,16 @@ else()
...
+    add_cxx_compiler_flag(-wd2259)
+    # 2. Function declared but never referenced (sysinfo.cc)
+    add_cxx_compiler_flag(-wd177)    
+    # 3. Unrecognized #pragma (gtest-port.cc)
```
Comment (LebedevRI) [inline CMakeLists.txt]: this is googletest issue

**Comment by rainergericke** [comment]
Hi Dominik,

I am sorry about the CLA bot problem but I cannot solve it by myself. It seems to be related to some mismatch with email addresses. Can you help me some how with it?

Best

Rainer

File: CMakeLists.txt (hunk 2 above)
Comment (dominichamon) [reply CMakeLists.txt]: agreed. but given it affects us is it a problem to disable this warning?

Comment (LebedevRI) [reply CMakeLists.txt]: not really, but this should ideally reference the upstream issue/pr with the same diff

**Comment by dominichamon** [comment]
i didn't see that the CLA issue got resolved. i think if we can change the tabs for spaces we can land this, right?

**Comment by LebedevRI** [comment]
reverse-ping, thanks

**Comment by dominichamon** [comment]
i think this was never completed. was it still an issue for folks?

**Pull request closed without merging**